autogen/main/variables.tf.tmpl

```diff
@@ -645,33 +645,36 @@ variable "timeouts" {
     error_message = "Only create, update, delete timeouts can be specified."
   }
 }
-{% if beta_cluster %}
-  {% if autopilot_cluster != true %}
 
-variable "enable_kubernetes_alpha" {
+{% if autopilot_cluster != true %}
+variable "monitoring_enable_managed_prometheus" {
   type        = bool
-  description = "Whether to enable Kubernetes Alpha features for this cluster. Note that when this option is enabled, the cluster cannot be upgraded and will be automatically deleted after 30 days."
+  description = "Configuration for Managed Service for Prometheus. Whether or not the managed collection is enabled."
   default     = false
 }
 
-variable "logging_enabled_components" {
+variable "monitoring_enabled_components" {
   type        = list(string)
-  description = "List of services to monitor: SYSTEM_COMPONENTS, WORKLOADS. Empty list is default GKE configuration."
+  description = "List of services to monitor: SYSTEM_COMPONENTS, WORKLOADS (provider version >= 3.89.0). Empty list is default GKE configuration."
   default     = []
 }
 
-variable "monitoring_enabled_components" {
+variable "logging_enabled_components" {
   type        = list(string)
-  description = "List of services to monitor: SYSTEM_COMPONENTS, WORKLOADS (provider version >= 3.89.0). Empty list is default GKE configuration."
+  description = "List of services to monitor: SYSTEM_COMPONENTS, WORKLOADS. Empty list is default GKE configuration."
   default     = []
 }
-  {% endif %}
+{% endif %}
 
-variable "monitoring_enable_managed_prometheus" {
+{% if beta_cluster %}
+  {% if autopilot_cluster != true %}
+
+variable "enable_kubernetes_alpha" {
```

Comment:
`enable_kubernetes_alpha` is not beta feature as per https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster.

I understand that this is not monitoring configuration, but since you're already on this, could you please also move this out of beta